core/cloudflow-streamlets/src/main/scala/cloudflow/streamlets/StreamletContext.scala

```diff
@@ -89,11 +89,11 @@ trait StreamletContext {
       .getOrElse(throw MountedPathUnavailableException(volumeMount))
 
   case class MountedPathUnavailableException(volumeMount: VolumeMount)
-      extends Exception(s"Mount path for Volume Mount named [${volumeMount.name}] is unavailable.")
+      extends Exception(s"Mount path for Volume Mount named '${volumeMount.name}' is unavailable.")
 
 }
 
 case class PortNotFoundException(port: StreamletPort, streamletDefinition: StreamletDefinition)
     extends Exception(
-      s"Streamlet port ${port.name} not found for ${streamletDefinition.appId} and streamlet ${streamletDefinition.streamletRef}"
+      s"Streamlet port '${port.name}' not found for application '${streamletDefinition.appId}' and streamlet '${streamletDefinition.streamletRef}'"
```

Comment:
Maybe it makes sense to add a `isOutlet` to `StreamletPort` (abstract), implement in `Inlet` and `Outlet`, or even add a `portType` to `StreamletPort` (abstract) and implement as `"inlet"` in `Inlet` and `"outlet"` in `Outlet` so we can be more precise in feedback to user..

Reply:
I agree, since the word "port" is not common for end-users and will get even more complex when we start adding gRPC and unmanaged connections. the more clear we can be in these errors the better is it.

That said, I'll merge this PR since I don't want to block it.